apluslms_roman/backends/__init__.py

```diff
@@ -98,8 +98,8 @@ class Backend:
     WORK_PATH = '/work'
     LABEL_PREFIX = 'io.github.apluslm.roman'
 
-    def __init__(self, environment: Environment):
-        self.environment = environment
+    def __init__(self, options: Environment):
+        self.options = options
```

Comment:
I kind of forgot where we ended up in the conversation, was this name okay...?

Reply:
`self.context = context` might be good, and then the value within would be `self.context.options` and `.self.context.uid` and so on.